packages/unity-bootstrap-theme/src/scss/extends/_list.scss

```diff
@@ -14,49 +18,107 @@
   }
 }
 
-@mixin uds-ul-list-styles {
-  li:before,
-  ul ul li:before,
-  ul ul ul ul li:before,
-  ul ul ul ul ul ul li:before,
-  ul ul ul ul ul ul ul ul li:before,
-  ul ul ul ul ul ul ul ul ul ul li:before {
-    content: '\2022 '; //\2022 ;
-    font-size: 2rem;
-    vertical-align: middle;
-    line-height: 1.5rem;
-    padding-right: 1.25rem; // Magic number.;
-    margin-left: -2rem;
-  }
+/* Apply CSS vars */
+ol, ul{
+  max-width: var(--list-max-width);
+  padding-top: var(--list-padding-top);
+  padding-right: var(--list-padding-right);
+  padding-bottom: var(--list-padding-bottom);
+  padding-left: var(--list-padding-left);
+  list-style: var(--list-list-style);
 
-  ul li:before,
-  ul ul ul li:before,
-  ul ul ul ul ul li:before,
-  ul ul ul ul ul ul ul li:before,
-  ul ul ul ul ul ul ul ul ul li:before {
-    content: '\25E6 '; //\2022 -empty;
+  li:before {
+    content: var(--li-before-content);
+    font-size: var(--li-before-font-size);
+    color: var(--li-before-color);
+    background-color: var(--li-before-background-color);
+    line-height: var(--li-before-line-height);
   }
+}
 
-  ol,
-  ul {
-    padding: 1rem 0 0 1.5rem;
-    list-style: none;
-  }
+/* Nested Lists */
+ol.uds-list ol,
+ul.uds-list ol,
+ol.uds-list ul,
+ul.uds-list ul {
+  --list-padding-top: 1rem;
+  --list-padding-right: 0;
+  --list-padding-bottom: 0;
+  --list-padding-left: 1.5rem;
+  --list-list-style: none;
 }
 
-// General UL rules.
-ul.uds-list {
+
+
+/* Nested Unordered Lists */
+// ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul>li>ul,
```

Comment:
Do we want to support nesting even further? I can uncomment or remove these extra nested items